### PR TITLE
Travis config

### PR DESCRIPTION
**Pull request by pq**
@keertip @devoncarew

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+language: dart
+script: ./tool/travis.sh
```
Comment (devoncarew) [inline .travis.yml:2]: If you don't need to do any sudo'ing (which you shouldn't), adding this line:

```
sudo: false
```

will use containerized builds on travis - the builds will start up much faster.

File: tool/travis.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+# Fast fail the script on failures.		
+set -e
+
+# Verify that the libraries are error free.
+dartanalyzer --fatal-warnings \
+  bin/analyzer.dart
```
Comment (devoncarew) [inline tool/travis.sh]: you may want to add `lib/src/analyzer_impl.dart` here as well - dartanalyzer way ignore issues in code referenced via package: imports.

**Comment by devoncarew** [comment]
lgtm

File: .travis.yml (hunk 1 above)
Comment (pq) [inline .travis.yml:2]: Great.  Thanks!  Fixed.

File: tool/travis.sh (hunk 2 above)
Comment (pq) [inline tool/travis.sh]: Done.  Soon I hope to make it "just work" for folders.  One that happens we can simplify this to ".".

**Pull request merged**